tests/queries/0_stateless/03034_normalized_ast.sql

```diff
@@ -0,0 +1,8 @@
+-- https://github.com/ClickHouse/ClickHouse/issues/49472
+SELECT
+    concat(database, table) AS name,
+    count()
+FROM clusterAllReplicas(default, system.tables)
```

Comment:
The `default` cluster doesn't exist anymore for tests ( https://github.com/ClickHouse/ClickHouse/pull/61396). You can use `test_shard_localhost` instead

Reply:
Thanks!